### ISSUE TITLE
puppet-agent: wrong repo url on ubuntu 18.04

### ISSUE THREAD

**Issue by brightdroid** [example42/tinydata#27]
## Expected Behavior 

add correct repo url `deb http://apt.puppetlabs.com bionic puppet`

## Actual Behavior

repo with url `deb http://apt.puppetlabs.com stable puppet5` is added

## Steps to Reproduce the Problem

  1. apply puppet code ```tp::install{'puppet-agent': }```

## Specifications

`facter -p` output:
```
{
  architecture => "amd64",
  distro => {
    codename => "bionic",
    description => "Ubuntu 18.04 LTS",
    id => "Ubuntu",
    release => {
      full => "18.04",
      major => "18.04"
    }
  },
  family => "Debian",
  hardware => "x86_64",
  name => "Ubuntu",
  release => {
    full => "18.04",
    major => "18.04"
  },
  selinux => {
    enabled => false
  }
}
```

**Comment by alvagante** [comment]
Should be fixed by the referenced commit.
Note that the current data for puppet-agent allows both the installation of the release package via dpkg (with tp > 2.2.0 (using tinydata keys **repo_package_name** and  **repo_package_url**) or, for apps for which this data is not available or when using older tp versions, configuring directly the sources list file and the public key (with the keys:  **repo_url, key, key_url, apt_repos, apt_release** ..).
I'm not fully sure that there's the correct data for all the os combinations .

**Comment by alvagante** [comment]
Note: this has to be merged https://github.com/example42/puppet-tp/pull/54 to fix an error with tp 2.2.0 used via command-line (or whenever a valid path is  not a resource default for exec type) when repo_package data is available
